### ISSUE TITLE
[Feature request] Ability to hide certain tab groups and not others

### ISSUE THREAD

**Issue by FranciscoMurias** [threethan/LightningLauncher#120]
This is listed as an option at the sidequest page, but currently you cannot hide individual tab groups while retaining others. This would be a good feature to have

**Comment by threethan** [comment]
I'm not entirely sure what you're requesting.
Would you like the option to remove certain app groups from the groups bar?
Or would you like to display multiple groups of apps at the same time? (App groups may be toggled individually by long-pressing on the group bar outside of edit mode)

**Comment by FranciscoMurias** [comment]
Yeah, the ability to remove (hide) certain app groups from the groups bar. The same way you can hide apps, adding the option to hide groups would be equally useful.

**Comment by threethan** [comment]
I've thought a bit about this. 

The ability to hide apps was originally added as a workaround for unlaunchable utility apps; I already find it a bit unintuitive for my liking. No real equivalent exists for groups. Showing and hiding groups would require going into edit mode to launch an app from a hidden category, which may confuse new users and doesn't fit in with the intended use of this app.
